openshift-install-powervs

```diff
@@ -1356,6 +1375,10 @@ function main {
       warn "Enabling forceful destruction option for terraform destroy command"
       FORCE_DESTROY=1
       ;;
+    "-force-run")
+      warn "Enabling forceful run option for executing utility."
```

Comment:
Hmm.. how about naming it as `no-platform-check` ?

Reply:
can be done @yussufsh your thoughts?

Reply:
how about `no-os-check`? Because we have error message at end saying...
```
Unsupported platform: ${PLATFORM}
```
where platforms could be Linux, Mac, Windows.

Or we just use -force.

Reply:
Resolved.